impeller/entity/contents/runtime_effect_contents.cc

```diff
@@ -206,7 +206,7 @@ bool RuntimeEffectContents::Render(const ContentContext& renderer,
     size_t buffer_offset = 0;
 
     for (const auto& uniform : runtime_stage_->GetUniforms()) {
-      std::shared_ptr<ShaderMetadata> metadata = MakeShaderMetadata(uniform);
+      ShaderMetadata metadata = MakeShaderMetadata(uniform);
```

Comment:
this doesn't need to be a shared_ptr since its going to be copied into the bindings object anyway.